### PR TITLE
MAB suspect ranges for CTDBP, CTDMO, CTDPF, DOSTA, and DOFST

### PR DESCRIPTION
**Pull request by reedan88**

### CODE REVIEW

File: qartod/ctdpf/ctdpf_qartod_gross_range_test_values.csv [hunk 1]
```diff
@@ -229,43 +229,43 @@ CP02PMUO,WFP01,03-CTDPFK000,ctdpf_ckl_wfp_instrument_recovered,{'inp': 'ctdpf_ck
...
 CP14NEPM,WFP01,03-CTDPFK000,ctdpf_ckl_wfp_instrument,{'inp': 'ctdpf_ckl_seawater_conductivity'},"{'qartod': {'gross_range_test': {'fail_span': [0, 9]}}}","Sensor min/max derived from vendor documentation or site depth, whichever is greater.",User range to be determined.
-CP14NEPM,WFP01,03-CTDPFK000,ctdpf_ckl_wfp_instrument_recovered,{'inp': 'practical_salinity'},"{'qartod': {'gross_range_test': {'fail_span': [0, 42]}}}","Sensor min/max derived from vendor documentation or site depth, whichever is greater.",User range to be determined.
+CP14NEPM,WFP01,03-CTDPFK000,ctdpf_ckl_wfp_instrument_recovered,{'inp': 'practical_salinity'},"{'qartod': {'gross_range_test': {'fail_span': [0, 42], 'suspect_span': [0, 406.9531]}}}","Sensor min/max derived from vendor documentation or site depth, whichever is greater.",The user range was determined from a characteristic profile for the location of the buoy based on World Ocean Database data. User range will be updated when 1 year of location specific data has been collected.
```
Comment (mvardaro) [inline qartod/ctdpf/ctdpf_qartod_gross_range_test_values.csv:260]: The upper "suspect_span" value here looks suspect (compared to the fail_span).

File: qartod/ctdpf/ctdpf_qartod_gross_range_test_values.csv [hunk 2]
```diff
@@ -229,43 +229,43 @@ CP02PMUO,WFP01,03-CTDPFK000,ctdpf_ckl_wfp_instrument_recovered,{'inp': 'ctdpf_ck
...
 CP13SOPM,WFP01,03-CTDPFK000,ctdpf_ckl_wfp_instrument_recovered,{'inp': 'ctdpf_ckl_seawater_conductivity'},"{'qartod': {'gross_range_test': {'fail_span': [0, 9]}}}","Sensor min/max derived from vendor documentation or site depth, whichever is greater.",User range to be determined.
-CP14NEPM,WFP01,03-CTDPFK000,ctdpf_ckl_wfp_instrument,{'inp': 'practical_salinity'},"{'qartod': {'gross_range_test': {'fail_span': [0, 42]}}}","Sensor min/max derived from vendor documentation or site depth, whichever is greater.",User range to be determined.
+CP14NEPM,WFP01,03-CTDPFK000,ctdpf_ckl_wfp_instrument,{'inp': 'practical_salinity'},"{'qartod': {'gross_range_test': {'fail_span': [0, 42], 'suspect_span': [0, 406.9531]}}}","Sensor min/max derived from vendor documentation or site depth, whichever is greater.",The user range was determined from a characteristic profile for the location of the buoy based on World Ocean Database data. User range will be updated when 1 year of location specific data has been collected.
```
Comment (mvardaro) [inline qartod/ctdpf/ctdpf_qartod_gross_range_test_values.csv:256]: The upper "suspect_span" value here looks suspect (compared to the fail_span).

File: qartod/ctdpf/ctdpf_qartod_gross_range_test_values.csv [hunk 3]
```diff
@@ -229,43 +229,43 @@ CP02PMUO,WFP01,03-CTDPFK000,ctdpf_ckl_wfp_instrument_recovered,{'inp': 'ctdpf_ck
...
 CP14SEPM,WFP01,03-CTDPFK000,ctdpf_ckl_wfp_instrument,{'inp': 'ctdpf_ckl_seawater_conductivity'},"{'qartod': {'gross_range_test': {'fail_span': [0, 9]}}}","Sensor min/max derived from vendor documentation or site depth, whichever is greater.",User range to be determined.
-CP14SEPM,WFP01,03-CTDPFK000,ctdpf_ckl_wfp_instrument_recovered,{'inp': 'practical_salinity'},"{'qartod': {'gross_range_test': {'fail_span': [0, 42]}}}","Sensor min/max derived from vendor documentation or site depth, whichever is greater.",User range to be determined.
+CP14SEPM,WFP01,03-CTDPFK000,ctdpf_ckl_wfp_instrument_recovered,{'inp': 'practical_salinity'},"{'qartod': {'gross_range_test': {'fail_span': [0, 42], 'suspect_span': [0, 406.9531]}}}","Sensor min/max derived from vendor documentation or site depth, whichever is greater.",The user range was determined from a characteristic profile for the location of the buoy based on World Ocean Database data. User range will be updated when 1 year of location specific data has been collected.
```
Comment (mvardaro) [inline qartod/ctdpf/ctdpf_qartod_gross_range_test_values.csv:268]: The upper "suspect_span" value here looks suspect (compared to the fail_span).

**Review by mvardaro — CHANGES_REQUESTED** [review]
Almost everything looks good, except for the NEPM and SEPM profiler CTDPF suspect_span values for salinity. The upper bounds look too high ('suspect_span': [0, 406.9531]) compared to the fail_span ('fail_span': [0, 42]).

**Review by cwingard — APPROVED** [review]
Really tight salinity range for those profilers. Does it really not vary that much there?

**Comment by reedan88** [comment]
@cwingard No, when I fixed the issue Mike pointed out, I accidentally created a _different_ issue. I fixed that up and now the ranges look much more like what I expect.

**Pull request merged**